CHANGELOG.md

```diff
@@ -1,12 +1,19 @@
 Changes by Version
 ==================
 
-0.36.0 (unreleased)
+1.1.0 (unreleased)
 -------------------
 
 - Nothing yet.
 
 
+1.0.0 (2019-08-02)
+-------------------
+
+- Support OpenTracing 0.32.0 and 0.33.0 ([#623](https://github.com/jaegertracing/jaeger-client-java/pull/623)) [@pavolloffay](https://github.com/pavolloffay)
```

Comment:
The stile is mist matched, also in the last release. It should be 

Support OpenTracing 0.32.0 and 0.33.0  ([#623](https://github.com/jaegertracing/jaeger-client-java/pull/623), [@pavolloffay](https://github.com/pavolloffay))